include/ginkgo/core/distributed/vector.hpp

```diff
@@ -404,6 +404,32 @@ class Vector
      */
     void compute_norm1(ptr_param<LinOp> result, array<char>& tmp) const;
 
+    /**
+     * Computes the column-wise mean of this (multi-)vector using a global
+     * reduction.
+     *
+     * @param result  a Dense row matrix, used to store the mean
+     *                (the number of columns in result must match the number
+     *                of columns of this)
+     * @param tmp  the temporary storage to use for partial sums during the
+     *             reduction computation. It may be resized and/or reset to the
+     *             correct executor.
```

Comment:
it does not have the parameter here

Reply:
Also, I guess, `ptr_param<LinOp> result` is the way to go here?